src/textual/css/scalar_animation.py

```diff
@@ -48,11 +52,21 @@ def __init__(
             assert duration is not None, "Duration expected to be non-None"
             self.duration = duration
 
+        self.show_animations = active_app.get().show_animations
```

Comment:
The animator has a reference to app, so I think we can avoid this by moving some of the logic to the animator.

Reply:
Removed it from here.